docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -101,7 +102,23 @@ The document is intended for readers with a good knowledge of Linux server manag
 The X-Road Security Server Toolkit package can be installed using PIP:
 
 ```
-$ pip install xrdsst --extra-index-url https://xroad-toolkit.s3-eu-west-1.amazonaws.com/xrdsst
+$ pip install -i http://xroad-toolkit.s3-website-eu-west-1.amazonaws.com/ xrdsst --trusted-host xroad-toolkit.s3-website-eu-west-1.amazonaws.com
+```
+
+** The public key ``jenkins@niis.org`` has been added to an SKS server: pool.sks-keyservers.net **
+
+The fingerprint of that key:
+```
+pub   rsa4096 2018-05-31 [SC]
+      BEC35825BBAB4288933F0354116AC90A8F670D74
+uid           [ unknown] Jenkins (X-Road Development Signing Key) <jenkins@niis.org>
+```
+
+The downloaded and signed packages can be verified with the public key:
+
+```
+$ gpg --keyserver pool.sks-keyservers.net --search-keys  BEC35825BBAB4288933F0354116AC90A8F670D74
+$ gpg --verify xrdsst-*.tar.gz.asc xrdsst-*.tar.gz 
```

Comment:
Hard-coded signature file name, so no need for wildcards anymore, they were there because of the version number, which can not be hard-coded, but now the signature file is renamed to have always the same name.